draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
+   <xref target="polling-collections-extended"/>.
    <vspace blankLines="1"/>
-   Value: One of the JSON values "true" (the matching is case
-         sensitive) or "false" (the matching is case insensitive).
+   Value: An array of JSON-encoded triggers.
    <vspace blankLines="1"/>
-   Mandatory: No; default is "false", i.e., a case-insensitive match.
+   Mandatory: No. The "all-triggers" SHOULD only be used by dCDN that supports and advertises 
+   the appropriate extended status for trigger collections (see <xref target="extended-status-capability-object"/> for details).
    </t>
 
-   <t hangText="Name: match-query-string">
+   <t hangText="Name: cdn-id">
    <vspace blankLines="1"/>
-   Description: Flag indicating whether to include the query part
-         of the URI when compared against the pattern.
+   Description: The CDN PID of dCDN.
    <vspace blankLines="1"/>
-   Value: One of the JSON values "true" (the full URI, including
-         the query part, should be compared against the given pattern)
-         or "false" (the query part of the URI should be dropped before
-         comparison with the given pattern).
+   Value: A JSON string, dCDN's CDN PID, as defined in <xref target="loop-detection-and-prevention"/>.
    <vspace blankLines="1"/>
-   Mandatory: No; default is "false". The query part of the URI
-         should be dropped before comparison with the given pattern.
+   Mandatory: Only in the collection of all triggers, if dCDN implements the filtered collections.
+         Optional in the filtered collections (uCDN can always find
+         dCDN's cdn-id in the collection of all triggers, but dCDN can choose to repeat that information
+         in its implementation of filtered collections).
    </t>
-        <t hangText="Name: url-type">
-        <vspace blankLines="1"/>
-        Description: Type of URLs to match.
-        <vspace blankLines="1"/>
-        Value: URL Type as defined in <xref target="url-types"/>.
-        <vspace blankLines="1"/>
-        Mandatory: No. When omitted or empty, "published" URL type is assumed.
-        </t>
-
 
    </list></t>
    </list></t>
 
-   <t>Example of case-sensitive prefix match against "https://www.example.com/trailers/":</t>
-<sourcecode type="json" name="prefix-match">
-<![CDATA[
-{
-    "pattern": "https://www.example.com/trailers/*",
-    "case-sensitive": true
-}
-]]></sourcecode>
-   </section>
-</section>
-
+      <section title="Filtered Collections by Status Link" anchor="filtered-collection-label-link">
+      <t>
+         The Filtered Collections by state Link object represents a link to a filtered collection
+	per <xref target="trigger-state">trigger state</xref>.
+         It is encoded as a JSON object containing the following name/value pairs:
+      </t>
+           <t><list style="empty" hangIndent="3">
+      <t><list style="hanging" hangIndent="3"><t hangText="Name: status">
```

Comment:
It seems like we should now call this "state" instead of "status" for consistency.  The example also still references "status".